test/integration/profiler_test.go

```diff
@@ -54,7 +55,7 @@ import (
 	"github.com/parca-dev/parca-agent/pkg/profile"
 	"github.com/parca-dev/parca-agent/pkg/profiler"
 	"github.com/parca-dev/parca-agent/pkg/profiler/cpu"
-	"github.com/parca-dev/parca-agent/pkg/runtime"
+	rn "github.com/parca-dev/parca-agent/pkg/runtime"
```

Comment:
```suggestion
	goruntime "github.com/parca-dev/parca-agent/pkg/runtime"
```
This is what we use in other places. Let's be consistent.